wp-content/civi-extensions/goonjcustom/Civi/CollectionCampService.php

```diff
@@ -83,6 +87,11 @@ public static function collectionCampTabset($tabsetName, &$tabs, $context) {
       "wp-admin/admin.php?page=CiviCRM&q=civicrm%2Fadmin-camp-outcome-form",
     );
 
+    // URL for the material dispatch authorizationtab.
+    $materialAuthorisation = \CRM_Utils_System::url(
```

Comment:
Use consistent naming
```suggestion
    $materialAuthorization = \CRM_Utils_System::url(
```